### ISSUE TITLE
Feature Request: completely support dark theme

### ISSUE THREAD

**Issue by spirritus** [josdejong/jsoneditor#1295]
Hi there,

your editor look good so far. An automatic theme detection would be awesome.
And dark versions of the various ui elements, i.e. sort and search.

Cheers

**Comment by josdejong** [comment]
Thanks for your suggestion, a dark theme would be nice indeed.

You can create a custom theme yourself, see: https://github.com/josdejong/jsoneditor/blob/develop/examples/06_custom_styling.html

**Comment by leafOfTree** [comment]
The dark style https://github.com/josdejong/jsoneditor/blob/develop/examples/06_custom_styling.html already looks pretty good, except the collapse/expand button icon not very clear. Maybe we could add extra icons to the svg for the dark theme.

<img width="766" alt="Screen Shot 2021-04-20 at 7 01 13 PM" src="https://user-images.githubusercontent.com/13349592/115386433-f6917580-a20b-11eb-8677-07f65a15b7f0.png">

**Comment by leafOfTree** [comment]
Hi, the https://github.com/josdejong/jsoneditor/blob/develop/examples/06_custom_styling.html already looks pretty good, except for the collapse/expand button icon. Maybe we could add extra icons to the [svg](https://github.com/josdejong/jsoneditor/blob/master/src/scss/img/jsoneditor-icons.svg) for the dark theme.

<img width="766" alt="Screen Shot 2021-04-20 at 7 01 13 PM" src="https://user-images.githubusercontent.com/13349592/115387093-c9919280-a20c-11eb-9876-79c3ab93ecf7.png">

**Comment by josdejong** [comment]
Yes indeed, a dark themed version of all icons would make it much prettier
